src/3_PB/PianoDiProgetto/sections/PianificazioneSprint/DodicesimoSprint.typ

```diff
@@ -11,4 +11,5 @@
     - Aggiornare pianificazione e preventivo pertinenti allo #glossary[sprint] 12 e inserire il consuntivo pertinente allo #glossary[sprint] 11;
     - Aggiornare ed aggiungere rispettivamente per gli #glossary[sprint] 13 e 14, pianificazione e preventivo;
   - Perfezionare il documento _Specifiche Tecniche v2.0_;
-  - Continuazione nello sviluppo del prodotto.
\ No newline at end of file
+  - Continuazione della codifica del prodotto e introduzione della componente di allarmistica. 
```

Comment:
```suggestion
  - Continuazione della codifica del prodotto e introduzione della componente di allarmistica;
```
volendo si potrebbe/dovrebbe essere più dettagliati negli obiettivi della pianificazione, specificando esattamente cosa si è andati a codificare (quasi task-level) sia in questo sprint che in quello prima: mi riferisco all'implementazione della nuova architettura e alla codifica dei vari sensori (che sensori abbiamo aggiunto di sprint in sprint?)

Reply:
manca la menzione delle modifiche apportate di recente alle NdP e anche l'inizio della stesura del Manuale Utente

Reply:
questo fatto